src/bot/games/gamble.ts

```diff
@@ -37,96 +37,150 @@ class Gamble extends Game {
   chanceToTriggerJackpot = 5;
   @shared(true)
   jackpotValue = 0;
+  @settings("timeout")
+  gambletimeout = 30
+
+  timeoutList: Dictionary<number> = {};
+
+  @command('!jackpot')
+  async jackpot (opts: CommandOptions): Promise<CommandResponse[]> {
+    let message;
+
+    if (this.enableJackpot) {
+      const currentPointsOfUser = await pointsSystem.getPointsOf(opts.sender.userId);
+      message = prepare('gambling.gamble.jackpotinfo', {
+        pointsName: await pointsSystem.getPointsName(currentPointsOfUser),
+        jackpotName: await pointsSystem.getPointsName(this.jackpotValue),
+        jackpot: Math.ceil(this.jackpotValue),
+      });
+
+      /*message = prepare('gambling.gamble.winJackpot', {
```

Comment:
Why it is commented and what is difference between winJackpot and jackpotinfo?

Reply:
The Commented line was only for me Temporaly an can be deletet ^^
i have forget them to delete... sry